frontend/src/main/components/UCSBSubjects/UCSBSubjectsTable.js

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+// import { toast } from "react-toastify";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, onDeleteSuccess } from "main/utils/UCSBSubjectUtils"
+import { useNavigate } from "react-router-dom";
+import { hasRole } from "main/utils/currentUser";
+
+export default function UCSBSubjectsTable({ subjects, currentUser }) {
+
+    const navigate = useNavigate();
+
+    const editCallback = (cell) => {
+        navigate(`/UCSBSubjects/edit/${cell.row.values.id}`)
+    }
+
+    // Stryker disable all : hard to test for query caching
+
+    const deleteMutation = useBackendMutation(
+        cellToAxiosParamsDelete,
+        { onSuccess: onDeleteSuccess },
+        ["/api/UCSBSubjects/all"]
+    );
+    // Stryker enable all 
+
+    // Stryker disable next-line all : TODO try to make a good test for this
+    const deleteCallback = async (cell) => { deleteMutation.mutate(cell); }
```

Comment:
We acknowledge the lack of coverage for this line. We will come back to it when we have some sort of understanding of what's going on.